bin/create_hugo_data.py

```diff
@@ -494,13 +494,126 @@ def export_events(anthology, builddir, dryrun):
         else:
             data["title"] = f"{anthology.venues[main_venue].name} ({data['year']})"
 
+        # Add talks data if available
+        if event.talks:
```

Comment:
Technically unnecessary, you can save an indentation level if you just iterate directly :)

Reply:
Thanks, fixed it.